pydantic/main.py

```diff
@@ -169,6 +169,9 @@ class BaseModel(metaclass=_model_construction.ModelMetaclass):
     This replaces `Model.__fields__` from Pydantic V1.
     """
 
+    __pydantic_setattr_handlers__: ClassVar[Dict[str, Callable[[BaseModel, Any], None]]]  # noqa: UP006
+    """__setattr__ handlers. Used to speed up __setattr__."""
```

Comment:
I think it's worth a bit more strength/detail in this comment, e.g.:
```suggestion
    """__setattr__ handlers. Memoizing the setattr handlers leads to a dramatic performance improvement in `__setattr__`"""
```
(that suggestion would probably need reformatting.)